pkg/cmd/issue/view/view.go

```diff
@@ -163,10 +165,14 @@ func findIssue(client *http.Client, baseRepoFn func() (ghrepo.Interface, error),
 	return issue, repo, err
 }
 
-func printRawIssuePreview(out io.Writer, issue *api.Issue) error {
-	assignees := issueAssigneeList(*issue)
-	labels := issueLabelList(issue, nil)
-	projects := issueProjectList(*issue)
+func rawIssuePreview(IO *iostreams.IOStreams, issue *api.Issue) error {
+
+	out := IO.Out
+
+	assignees := issue.GetAssigneeListString()
+	// Labels no longer have color in the raw issue preview
```

Comment:
This is a choice I made to clean up the code. Let me know if you agree or disagree. Notably, no tests failed when I made this change 😅